packages/error-boundary-fallback/src/components/ErrorBoundary/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Result, Button } from 'antd';
+import { useHistory } from 'react-router';
+
+const ErrorBoundary: React.FC = () => {
+  const history = useHistory();
+
+  return (
+    <Result
+      status="error"
+      title="An Error Occurred"
+      subTitle="There has been an error. It’s been reported to the site administrators via email and
+      should be fixed shortly. Thanks for your patience."
+      extra={
+        <Button id="backHome" key="error" onClick={() => history.push('/')} type="primary">
```

Comment:
would we want to support a scenario where the behaviour here is flexible i.e. if you want to redirect the user to a another page than `/` or if your home is not on the `/` route.